src/pages/Transactions/index.tsx

```diff
@@ -1,21 +1,20 @@
 import { useTranslation } from 'react-i18next';
-import { useSelector } from 'react-redux';
 
-import { StoreType } from '@/common/types/util.types';
 import ExternalLink from '@/components/ExternalLink';
 import { SUBSCAN_LINK } from '@/config/relay-chains';
 import MainContainer from '@/parts/MainContainer';
+import { useSubstrateSecureState } from '@/substrate-lib/substrate-context';
 
 import IssueRequestsTable from './IssueRequestsTable';
 import RedeemRequestsTable from './RedeemRequestsTable';
 
 const Transactions = (): JSX.Element => {
-  const { address } = useSelector((state: StoreType) => state.general);
+  const { selectedAccount } = useSubstrateSecureState();
   const { t } = useTranslation();
 
   return (
     <MainContainer>
-      <ExternalLink href={`${SUBSCAN_LINK}/account/${address}`} className='font-medium'>
+      <ExternalLink href={`${SUBSCAN_LINK}/account/${selectedAccount?.address ?? ''}`} className='font-medium'>
```

Comment:
So, when there is not account, do we still want to display this external link?

Reply:
Done!